lib/CrashHandler/CrashHandler.cpp

```diff
@@ -646,6 +792,15 @@ void CrashHandler::installCrashHandler() {
     ::alternativeStackMemory.release();
   }
 
+  try {
+    // Allocate the preallocated backtrace buffer
+    ::backtraceBuffer = std::make_unique<char[]>(::backtraceBufferSize);
+  } catch (...) {
+    // could not allocate memory for backtrace buffer.
+    // this is not critical - we can still work without it
+    ::backtraceBuffer.release();
+  }
```

Comment:
Maybe add a log message? Seems unlikely, but may confuse us very much without one if it does happen.